custom-elements.json

```diff
@@ -24,12 +24,23 @@
               "default": "'light'",
               "privacy": "public",
               "attribute": "theme"
+            },
+            {
+              "kind": "field",
+              "name": "config",
+              "default": "config",
+              "privacy": "public",
+              "attribute": "config"
             }
           ],
           "attributes": [
             {
               "name": "theme",
               "fieldName": "theme"
+            },
+            {
+              "name": "config",
+              "fieldName": "config"
             }
```

Comment:
Yeah, good catch. I've removed it from `properties`. According to [this](https://github.com/open-wc/custom-elements-manifest/issues/111) though I don't think it possible to remove it completely.

Sounds like even if we used the [private properties syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties) (which maybe we should) it would still appearing in `custom-elements.json`